src/test/kotlin/pt/up/fe/ni/website/backend/utils/documentation/payloadschemas/model/PayloadActivity.kt

```diff
@@ -13,6 +13,11 @@ class PayloadActivity {
                 DocumentedJSONField("id", "Id of the activity", JsonFieldType.NUMBER),
                 DocumentedJSONField("title", "Title of the activity", JsonFieldType.STRING),
                 DocumentedJSONField("description", "Description of the activity", JsonFieldType.STRING),
+                DocumentedJSONField(
+                    "hallOfFame",
+                    "Array of members that were once associated with the project",
+                    JsonFieldType.ARRAY
+                ),
```

Comment:
```suggestion
```
not part of activity

Reply:
This was because Spring includes project/event's properties when serializing activities. This is unexpected but shouldn't be a problem